homeassistant/components/media_player/xiaomi_tv.py

```diff
@@ -0,0 +1,112 @@
+"""
+Add support for the Xiaomi TVs.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/xiaomi_tv/
+"""
+
+import logging
+import voluptuous as vol
```

Comment:
Add a blank line between standard library and 3rd party imports.

Reply:
Just looked at another platform, and you're absolutely right. Will update in next pull request.